### ISSUE TITLE
Call to a member function getStatusCode() on integer in Slim/App.php on line 354

### ISSUE THREAD

**Issue by xtreamwayz** [slimphp/Twig-View#8]
For some reason Twig->render() doesn't return a response. What does work for me is this:

```
    public function render(ResponseInterface $response, $template, $data = [])
    {
       $response->getBody()->write($this->fetch($template, $data));

        return $response;
    }
```

**Comment by xtreamwayz** [comment]
The problem is that Twig->render() uses $response->getBody(). It returns the body. If you write directly to the response it fixes this as well:

```
    public function render(ResponseInterface $response, $template, $data = [])
    {
        return $response->write($this->fetch($template, $data));
    }
```

To clarify what I did: I used the example from http://docs-new.slimframework.com/features/templates/ -> Use the view service

**Comment by akrabat** [comment]
Fixed by PR #5.
